README.md

```diff
@@ -29,16 +26,17 @@ Make sure to use these settings:
 * **Logzio Host**: Use your Logz.io region's listener URL.
   If your login URL is app.logz.io, use `listener.logz.io` (this is the default setting).
   If your login URL is app-eu.logz.io, use `listener-eu.logz.io`.
-* **Logzio Token**: Use the [token](https://app.logz.io/#/dashboard/settings/general) of the account you want to ship to.
+* **Logs Account Token**: Use the [token](https://app.logz.io/#/dashboard/settings/general) of the logs account you want to ship to.
+* **Metrics Account Token**: Use the [token](https://app.logz.io/#/dashboard/settings/general) of the metrics account you want to ship to.
 
 At the bottom of the page, select **I agree to the terms and conditions stated above**, and then click **Purchase** to deploy.
 
 Deployment can take a few minutes.
 
-### 3. _(Optional)_ Add failsafe for log shipping timeouts
+### 3. _(Optional)_ Add failsafes for shipping timeouts
 
-You can configure logzio-azure-serverless to back up logs to Azure Blob Storage.
-So if the connection to Logz.io times out or an error occurs, you'll still have a backup of any dropped logs.
+You can configure logzio-azure-serverless to back up logs and metrics to Azure Blob Storage.
+So if the connection to Logz.io times out or an error occurs, you'll still have a backup of any dropped data that didn't get shipped.
 
 ![Function app left menu](img/function-app-menu-integrate.png)
```

Comment:
@imnotashrimp 
The name of the functions changes so the image is not good anymore. For now, let's take in off(img/function-app-menu-integrate.png).

Reply:
done